ogre/include/ignition/rendering/ogre/OgreThermalCamera.hh

```diff
@@ -112,6 +117,7 @@ namespace ignition
       private: std::unique_ptr<OgreThermalCameraPrivate> dataPtr;
 
       private: friend class OgreScene;
+      private: friend class OgreRayQuery;
```

Comment:
is the `friend class` still needed now that it's using `OgreObjectInterface`? Same goes for other ogre camera headers

Reply:
You're right - they're not needed now. Originally I had the new members as protected but changed them to public to match the access level of `OgreCamera()` in `Ogre2Camera` etc. I'll remove all the friend classes that are no longer required.